misc/services/mobile-webui/mobile-webui-frontend/src/apps/huManager/containers/HUBulkActionsScreen.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory, useRouteMatch } from 'react-router-dom';
+
+import { trl } from '../../../utils/translations';
+import { getHandlingUnitInfoFromGlobalState } from '../reducers';
+import { pushHeaderEntry } from '../../../actions/HeaderActions';
+import ButtonWithIndicator from '../../../components/buttons/ButtonWithIndicator';
+import BarcodeScannerComponent from '../../../components/BarcodeScannerComponent';
+import { toQRCodeDisplayable } from '../../../utils/huQRCodes';
+import * as api from '../api';
+import { toastError } from '../../../utils/toast';
+
+const HUBulkActionsScreen = () => {
+  const dispatch = useDispatch();
+  const history = useHistory();
+
+  const handlingUnitInfo = useSelector((state) => getHandlingUnitInfoFromGlobalState(state));
+
+  const [scannedHandlingUnitQRCodes, setScannedHandlingUnitQRCodes] = useState([handlingUnitInfo.qrCode.code]);
+  const [showHUScanner, setShowHUScanner] = useState(false);
+  const [showTargetScanner, setShowTargetScanner] = useState(false);
+
+  const { url } = useRouteMatch();
+  useEffect(() => {
+    if (!handlingUnitInfo) {
+      history.goBack();
+      return;
+    }
+
+    dispatch(pushHeaderEntry({ location: url, caption: trl('huManager.action.bulkActions.windowName') }));
+  }, []);
+
+  const onScannedHUResolved = ({ scannedBarcode }) => {
+    setScannedHandlingUnitQRCodes([...scannedHandlingUnitQRCodes, scannedBarcode]);
```

Comment:
check if the scannedBarcode already exists in scannedHandlingUnitQRCodes array and if yes, don't add it twice

Reply:
auch, forgot to add that back, I was testing adding a lot of HUs in the screen to see how the overlay is handled by default :)